.config/flakebox/.gitignore

```diff
@@ -0,0 +1 @@
+tmp/
```

Comment:
Why do we need that? Will this directory be created now or is it something local that could live in `.git/info/exclude`?

Reply:
Oh, that's in flakebox, nevermind